helm/values.yaml

```diff
@@ -70,18 +70,20 @@ ruler:
   ruleNamespaceSelector: {}
   ruleSelector: {}
   sinks:
-    alertmanagers:
-    - namespace: kubesphere-monitoring-system
-      name: alertmanager-operated
-    webhooks: []
+    # alertmanagers: []
+    # - namespace: kubesphere-monitoring-system
+    #   name: alertmanager-operated
```

Comment:
It may be better to modify it as follows: 
```
    alertmanager: 
      namespace: kubesphere-monitoring-system
      name: alertmanager-operated
    # alertmanagers: 
    # - namespace: kubesphere-monitoring-system
    #   name: alertmanager-operated
```

Reply:
Applied, PTAL